src/gateway/kong-plugins/authentication/oidc/okta.md

```diff
@@ -196,6 +189,17 @@ curl -i -X POST https://KONG_ADMIN_URL/routes/ROUTE_ID/plugins \
   --data config.scopes="profile"
```

Comment:
@rajeevkong I think we might be missing config.scopes_claim in this code block.

Reply:
We might also be missing `auth_method`? Not sure if anything else is missing

Reply:
@cloudjumpercat - I think you are right. I am sorry I have tried the API way to set up the plugin but I would assume we follow the same process as we do in the UI.

Reply:
@cloudjumpercat I committed to the same branch. Let me know if that doesnt work. Apologies for that.

Reply:
I merged that branch in, this should be ready to go.